qa/tasks/cephfs/test_volume_client.py

```diff
@@ -14,15 +14,11 @@ class TestVolumeClient(CephFSTestCase):
     # One for looking at the global filesystem, one for being
     # the VolumeClient, two for mounting the created shares
     CLIENTS_REQUIRED = 4
-    default_py_version = 'python3'
 
     def setUp(self):
         CephFSTestCase.setUp(self)
-        self.py_version = self.ctx.config.get('overrides', {}).\
-                          get('python3', TestVolumeClient.default_py_version)
-        log.info("using python version: {python_version}".format(
-            python_version=self.py_version
-        ))
+        # Only the python3 is support for the src/pybind.
+        self.py_version = 'python3'
```

Comment:
probably we can just remove this member variable as well?

Reply:
Yeah, makes sense.